README.md

```diff
@@ -15,7 +15,7 @@ Vue.js binding for firebase cloud firestore.
 
 ### Prerequisites
 
-Firebase ^5.0.0
+Firebase `5.0.0`
```

Comment:
The `^` has been removed? I have created PR #45 to add this back in

Reply:
Thanks @eddiejaoude